static/handler_test.go

```diff
@@ -156,11 +158,159 @@ var _ = Describe("func FromPackages() (handler analysis)", func() {
 				},
 			))
 		})
+		When("a handler is a type alias", func() {
+			BeforeEach(func() {
+				// Set the GODEBUG environment variable to enable type alias
+				// support.
+				//
+				// TODO: Remove this setting once the package is
+				// migrated to Go 1.23 that generates `types.Alias` types by
+				// default.
+				os.Setenv("GODEBUG", "gotypesalias=1")
+			})
+
+			AfterEach(func() {
+				os.Setenv("GODEBUG", "")
+			})
+
+			It("returns a single configuration for each handler type", func() {
+				cfg := packages.Config{
+					Mode: ConfigMode,
+					Dir:  "testdata/handlers/typealias",
+				}
+
+				pkgs := loadPackages(cfg)
+
+				apps := FromPackages(pkgs)
+				Expect(apps).To(HaveLen(1))
+				Expect(apps[0].Handlers().Aggregates()).To(HaveLen(1))
+				Expect(apps[0].Handlers().Processes()).To(HaveLen(1))
+				Expect(apps[0].Handlers().Projections()).To(HaveLen(1))
+				Expect(apps[0].Handlers().Integrations()).To(HaveLen(1))
+
+				aggregate := apps[0].Handlers().Aggregates()[0]
+				Expect(aggregate.Identity()).To(
+					Equal(
+						configkit.Identity{
+							Name: "<aggregate>",
+							Key:  "92623de9-c9cf-42f3-8338-33c50eeb06fb",
+						},
+					),
+				)
+				Expect(aggregate.TypeName()).To(
+					Equal(
+						"github.com/dogmatiq/configkit/static/testdata/handlers/typealias.AggregateHandler",
```

Comment:
Can we / do you think we should have this return the alias name instead?

Reply:
We can, lemme change that.